src/program/snabbnfv/neutron2snabb/neutron2snabb.lua

```diff
@@ -96,8 +96,13 @@ function filter (port, secbindings, secrules, direction)
          end
       end
    end
-   if #rules > 0 then return { rules = rules }
-                 else return nil end
+   if type == "stateless" then
```

Comment:
Does this change mean that we now include a PacketFilter app in the app network, even if no rules are defined?

That would be nice in principle - consistency - but in practice I wonder if this is a potential performance regression when no filtering is defined, which is the most important case in practice (e.g. for hosting router-like devices).

(Is there also a potential functional regression e.g. in the difference between no filtering (nil) vs no rules ({})? I am not sure.)